src/main/resources/public/js/app.js

```diff
@@ -9,11 +9,20 @@ $( document ).ready(function() {
    });
 });
 
-
 var state = "fire";
 var lastClickAcross;
 var lastClickDown;
 
+function startGame(mode) {
+	if (mode == "hard") {
+		
```

Comment:
I'll double check the JS and figure out if this happens before or otherwise. I can try to make JS changes as necessary.